package.json

```diff
@@ -26,9 +26,9 @@
   },
   "homepage": "https://github.com/companieshouse/restricted-word-web#readme",
   "dependencies": {
-    "axios": "~0.19.2",
+    "axios": "~0.21.1",
     "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#1.0.0",
-    "ch-structured-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#master",
+    "ch-structured-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#15a8112",
```

Comment:
Looking at other projects (e.g. https://github.com/companieshouse/strike-off-objections-web/blob/master/package.json) they use a version number not commit reference (which seems better)

    "@companieshouse/structured-logging-node": "^1.0.4",
    
    Same in https://github.com/companieshouse/dissolution-web/blob/master/package.json

Reply:
Because it requires that commit reference and it doesn't have a version